src/Apps/Artist/ArtistApp.tsx

```diff
@@ -87,6 +137,15 @@ export const ArtistApp: React.FC<ArtistAppProps> = props => {
   )
 }
 
+export const StyledLink = styled(RouterLink)`
```

Comment:
This gets re-used throughout the artist app, which is why I chose to export it from here.

We use `RouterLink` so we can avoid full page refreshes when moving between different sub-artist-pages.